lib/chatBot.js

```diff
@@ -61,7 +63,21 @@ var wcfg = { //define some extra logLevels for winston, also colors
 var ChatBot = function(username, password, options) {
 	var that = this;
 
-	this.version = ver;
+	require('gift')('.').status(function(err,status){
+		if(err || ver.indexOf("dev")===-1) {
+			that.version = ver;
+		} else {
+			var git = require('git-rev-sync');
+			that.version  = ver.replace('-dev','-git ');
+			that.version += git.branch()+"/"+git.short();
+			if(!status.clean) that.version+="+";
+		}
+	});
+	if(username instanceof Object) {
+		options = username;
+		if(options.username) username = options.username;
```

Comment:
![Codacy](https://www.codacy.com/versioned/images/favicon.png) Issue found: [Expected '{' and instead saw 'password'.](https://www.codacy.com/app/node-steam-chat-bot/steam-chat-bot/file/944346627/issues/source?bid=1980036&fileBranchId=2848073#l79)